src/VersionTool/VersionTool.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.0.0" />
+    <PackageReference Include="Microsoft.DotNet.Cli.Utils" Version="1.0.0-*" />
```

Comment:
Let's pin this to the RTM version (https://www.nuget.org/packages/Microsoft.DotNet.Cli.Utils/1.0.0-preview2-003121). We know it works with 1.0.0 NETCore.App